examples/coffeeshop.html

```diff
@@ -578,10 +578,27 @@ <H3 STYLE="flex-grow:1" ID="title">The CoffeeShop</H3>
       els.sample.contentDocument.body.style = "margin:0; padding:0;";
       postRun();
       els.sample.contentWindow.addEventListener('error', function(e) {
+        //console.log(e);
+        var rowOffset = 0;
+        if (e.filename == "") {
+          // Figure out line where `Algebra(..., ()=>{...}` starts in ACE Editor (because error is offsetted from there)
+          var lines = editor.getValue().split("\n");
+          rowOffset = -1;
+          for (var line of lines) {
+              if (line.match(/Algebra\s?\(/)) {
+                break;
+              }
+              rowOffset++;
+          }
+          if (rowOffset == -1) {
+            rowOffset = 0;
+          }
+        }
         // show error in HTML
         els.sample.contentDocument.body.innerHTML+=`<PRE>${e.message}</PRE>`;
         // show error in ace editor
-        var row = e.lineno;
+        var row = rowOffset + e.lineno;
+        //console.log("rowOffset", rowOffset, "e.lineno", e.lineno)
```

Comment:
I assume the commented-out line was not meant to be included in the PR, right?

Reply:
Thanks a lot, I could even remove more code for the Firefox/Chrome error messages... it was from before I figured out what happened with the error lines in different cases (inline code with offsetting from 0 etc.)